test/percy/spec.js

```diff
@@ -100,14 +103,20 @@ const Props = props => {
 Props.displayName = 'Props';
 Props.propTypes = {
   children: PropTypes.node.isRequired,
+  listPropsOfChild: PropTypes.bool.isRequired,
```

Comment:
Nit: How about `listPropsOfNestedChild`? Because this is a prop of `Spec`, and we're referring not to its child, but to the child of its child 👶 So it's like further _nested_  😃🤷‍♂

Reply:
ya that makes sense, switched it.